### PR TITLE
Build instructions converted to Markdown + update reference in README to them

### PR DESCRIPTION
**Pull request by Ridten**

### CODE REVIEW

**Comment by Ridten** [comment]
Linux instruction (Libraries part) a little bit dirty

File: doc/howto/HowToBuild-Linux.md [hunk 1]
```diff
@@ -1,95 +1,141 @@
...
+
 cmake ..
+
 make -jX # where X is the number of cores in your system
 ```
 
-#To enable debugging:
+## To enable debugging:
+
 cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo # Building with Debug flag isn't supported now
```
Comment (drug007) [inline doc/howto/HowToBuild-Linux.md:45]: `Release with debug symbols` mode allows to use debugger without performance degradation but most of error checks are not performed
```
cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
```
`Debug` mode allows to use debugger and enables all error checks causing huge performance degradation
```
cmake -DCMAKE_BUILD_TYPE=Debug ..
```
Note: `..` shall be the last argument of `cmake` call

File: doc/howto/HowToBuild-Linux.md [hunk 2]
```diff
@@ -1,95 +1,141 @@
...
-------
+## Setup:
+
 - Install S.T.A.L.K.E.R Call of Pripyat using wine (or obtain game resources another way, e.g. via Steam or GOG, all path below for example with wine).
-For example if you have a cd-rom you can type `wine setup.exe` to start installation. 
-It's recommended do not use path with spaces, for instance you can use "c:\cop" as 
+
+For example if you have a cd-rom you can type `wine setup.exe` to start installation.
+
+It's recommended do not use path with spaces, for instance you can use "c:\cop" as
```
Comment (drug007) [inline doc/howto/HowToBuild-Linux.md:111]: `Having spaces in path is not recommended, for instance you can use "c:\cop" as`

File: doc/howto/HowToBuild-Linux.md [hunk 3]
```diff
@@ -1,95 +1,141 @@
...
+
 an installation path (it will be `~/.wine/drive_c/cop` path on your host linux machine).
 
 After building in the engine folder do (assuming you installed CoP in "c:\cop"):
```
Comment (drug007) [inline doc/howto/HowToBuild-Linux.md:115]: `engine folder` -> `engine directory`

**Pull request closed without merging**